avatica/src/main/java/org/apache/calcite/avatica/proto/Requests.java

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
I think we talked about this before the protobuf changes got pulled in and there weren't strong feelings one way or the other. We could generate them at build time; the downside is that it forces the user (or build system) to have the protobuf compiler installed. Perhaps revive the old mail thread if you feel strongly?